lib/services/externalcloudaudit.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package services
+
+import (
+	"context"
+
+	"github.com/gravitational/trace"
+
+	externalcloudauditclient "github.com/gravitational/teleport/api/client/externalcloudaudit"
+	"github.com/gravitational/teleport/api/types/externalcloudaudit"
+	"github.com/gravitational/teleport/lib/utils"
+)
+
+var _ ExternalCloudAudit = (*externalcloudauditclient.Client)(nil)
+
+// ExternalCloudAuditGetter defines an interface for reading external cloud audits.
+type ExternalCloudAuditGetter interface {
+	// GetDraftExternalCloudAudit returns the draft external cloud audit resource.
+	GetDraftExternalCloudAudit(context.Context) (*externalcloudaudit.ExternalCloudAudit, error)
+	// GetClusterExternalCloudAudit returns the cluster external cloud audit resource.
+	GetClusterExternalCloudAudit(context.Context) (*externalcloudaudit.ExternalCloudAudit, error)
+}
+
+// ExternalCloudAudit defines an interface for managing ExternalCloudAudit.
+type ExternalCloudAudit interface {
```

Comment:
We already have an abundance of `ExternalCloudAudit` types, and while they are all justified, I think that this one in particular may cause some confusion, since it's not a resource, but a functional interface. I would even ask for something more expressive, but since I see that other files around use plural forms as a convention for interfaces that manage given types of resources, how about `ExternalCloudAudits`?

Reply:
renamed to `ExternalCloudAudits`